RELEASE-NOTES.md

```diff
@@ -17,12 +17,12 @@ Packages released:
 ## Chore
 - **A11y:** Ensure SkipToContent sends focus to page content ([#2058](https://github.com/patternfly/patternfly-next/pull/2125))
 - **Build:**
```

Comment:
This could change to `**Build: moved to circle ci`, then we could remove the "Moved to circleci. This PR:" line. I don't feel strongly about this, it's fine as is, too.